examples/plotting/time_distance.py

```diff
@@ -139,7 +140,7 @@
 # Plot the reference submap, line and extracted data from both methods and
 # the difference between them.
 
-fig = plt.figure(figsize=(8, 5), layout="constrained")
+fig = plt.figure(figsize=(10, 10), layout="constrained")
```

Comment:
Not sure how I feel about this change.

Reply:
yeah it's not as neat as it was before, maybe just make it slightly wider than it was before to accommodate the colour bars?

Reply:
Changed.